packages/react/spec/auto/hooks/useTable.spec.tsx

```diff
@@ -759,12 +812,18 @@ const loadMockWidgetModelMetadataForRelationship = () => {
                       apiIdentifier: "id",
                       fieldType: "ID",
                       __typename: "GadgetModelField",
+                      configuration: {
```

Comment:
Why do we care about the field config?

Reply:
We only care about the relationship config, but I used the `__typename` from the field config to determine whether the field is a relationship. Maybe I can just look at the field type instead.

Reply:
I think we already rely on the field type so it might be better to just couple ourselves to one thing -- I think the configs are a bit more likely to change.